resources/assets/lib/stores/channel-store.ts

```diff
@@ -18,7 +18,7 @@ import Message from 'models/chat/message';
 import core from 'osu-core-singleton';
 import UserStore from './user-store';
 
-const skippedChannelTypes = new Set<ChannelType>(['MULTIPLAYER', 'TEMPORARY']);
+const visibleChannelTypes = new Set<ChannelType>(['PM', 'PUBLIC']);
```

Comment:
I don't think we use it yet, but `GROUP` type should probably also be displayed.